core/new-gui/src/app/workspace/component/navigation/navigation.component.ts

```diff
@@ -325,6 +332,19 @@ export class NavigationComponent implements OnInit {
     }
   }
 
+  public onClickDisableOperators(): void {
```

Comment:
it will be good to include some docstring and comments. 

What happens if the highlights contain both disabled and undisabled operators and I click "disable"?